homeassistant/components/totalconnect/quality_scale.yaml

```
@@ -0,0 +1,65 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: todo
+  test-before-setup: todo
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Total Connect entities do not subscribe to events.
```

```suggestion
      Total Connect entities do not explicitly subscribe to events.
```
They do by using the coordinator

---

Thanks, I didn't catch that.  set to `todo`